src/ServiceControl.Audit.Persistence.RavenDb5/Indexes/MessagesViewIndex.cs

```diff
@@ -34,7 +33,7 @@ public MessagesViewIndex()
 
             Index(x => x.Query, FieldIndexing.Search);
 
-            Analyze(x => x.Query, typeof(StandardAnalyzer).AssemblyQualifiedName);
+            Analyze(x => x.Query, "StandardAnalyzer");
```

Comment:
We were including the entire Lucene package just to get one string. I believe this should work and be "the same" in RavenDB (not cause a re-index on deploy) but need to double-check.

Reply:
```suggestion
            Analyze(x => x.Query, "StandardAnalyzer"); // Not using typeof() to prevent dependency on Lucene
```